store/index.js

```diff
@@ -1,7 +1,17 @@
+import feathersVuex, { initAuth } from 'feathers-vuex'
+import feathersClient from '../feathers-client'
+
+const { service } = feathersVuex(feathersClient, { idField: '_id' })
+
+
 export const actions = {
   async nuxtServerInit ({dispatch}) {
     dispatch('categories/init')
     await dispatch('auth/init')
     await dispatch('settings/init')
   }
 }
+
+export const plugins = [
+  service('usersettings')
```

Comment:
calling `this.$store.dispatch('usersettings/find', {})` will run a number of mutations and switches `isFindPending` automagically: https://feathers-plus.github.io/v1/feathers-vuex/service-module.html